cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -484,14 +484,10 @@ func (c serviceAccountTokenControllerStarter) startServiceAccountTokenController
 		glog.Warningf("%q is disabled because there is no private key", saTokenControllerName)
 		return false, nil
 	}
-	privateKey, err := certutil.PrivateKeyFromFile(ctx.ComponentConfig.KubeCloudShared.ServiceAccountKeyFile)
-	if err != nil {
-		return true, fmt.Errorf("error reading key for service account token controller: %v", err)
-	}
 
 	var rootCA []byte
 	if ctx.ComponentConfig.SAController.RootCAFile != "" {
-		rootCA, err = ioutil.ReadFile(ctx.ComponentConfig.SAController.RootCAFile)
+		rootCA, err := ioutil.ReadFile(ctx.ComponentConfig.SAController.RootCAFile)
```

Comment:
does this shadow rootCA?

Reply:
@liggitt yes, it will. Thanks for your review